File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    do
    {
        printf("Height : ");
        n=get_int();
    }
    while ((n<0) || (n>23));
    {
        for (int i=0;i<n;i++)
        {
            int j=(n-1-i);
            while (j>0)
            {
                printf(" ");
                j=j-1;
            }
            int k=(i+1);
            while (k>0)
            {
                printf("#");
                k=k-1;
            }
            printf("  ");
            int l = i+1;
            while (l>0)
            {
                printf("#");
                l=l-1;
            }
            printf("\n");
        }
    }
    
}

File: mario.py
while True:
    try:
        print("Height: ",end="")
        a=int(input())
        if a>=0 and a<24:
            break
    except:
        pass
for i in range(a):
    print(" "*(a-i-1),end="")
    print("#"*(i+1),end="")
    print("  ",end="")
    print("#"*(i+1))